eZ/Publish/Core/Repository/Values/User/User.php

```diff
@@ -55,6 +55,14 @@ public function getVersionInfo()
     }
 
     /**
+     * {@inheritdoc}
+     */
+    public function getName($languageCode = null)
+    {
+        return $this->content->getVersionInfo()->getName($languageCode);
```

Comment:
same reply as above, convention in other method is to use real properties on same class directly.